setup.py

```diff
@@ -41,6 +41,7 @@
 install_requires = [
     "Flask~=1.1.2",
     "webargs~=5.5.0",
+    "speaklater>=1.3,<2.0",
```

Comment:
The pypi package/code is not maintained since 2012... Maybe is still okay, just pointing out possible "need to take over" in the future if vulns or similar appear.

Reply:
Thanks for that. I was initially thinking: "well, flask-babelex [which we use](https://github.com/inveniosoftware/invenio-i18n/blob/master/setup.py#L44) already depends [on it](https://github.com/mrjoes/flask-babelex/blob/master/setup.py#L44)". But clearly, flask-babelex is deprecated in favour of [flask-babel](https://github.com/python-babel/flask-babel) which [vendors in](https://github.com/python-babel/flask-babel/blob/master/flask_babel/speaklater.py) their own speaklater with their own LazyString class. Their version doesn't include the only function we use from it (is_lazy_string) though. 

So... we need to change our dependencies from flask-babelex in other invenio packages to flask-babel and make our own utility for lazy string determination (otherwise it won't work). Or we continue using speaklater like here because this is what the rest of invenio is using currently and change it later. Easiest is probably to go with option 2, but maybe it's better to fix it now? Bringing it to @lnielsen .